### PR TITLE
Doxygen improvements for sorc/grid_tools.fd/shave.fd/shave_nc.F90

### PR DESCRIPTION
**Pull request by lgannoaa**
PR for shave_nc.F90 doxygen

Part of #366

### CODE REVIEW

File: sorc/grid_tools.fd/shave.fd/shave_nc.F90 [hunk 1]
```diff
@@ -8,23 +8,28 @@
 !! the halo region. Then before the files go into the chgres
 !! job their domains must be shaved down to only the computational
 !! interior and the halo which is what this code does.
-!-----------------------------------------------------------------------
+!!
```
Comment (edwardhartnett) [inline sorc/grid_tools.fd/shave.fd/shave_nc.F90:11]: Author tag please.

**Comment by edwardhartnett** [comment]
@lgannoaa is this PR complete?

Please complete this PR so we can get it merged.

**Comment by edwardhartnett** [comment]
@lgannoaa please properly label each PR and put the issue addressed or fixed in the description. When I look at your open PRs, they all have the same title.

It seems like I have asked this several times. Please fix immediately.

**Comment by lgannoaa** [comment]
> @lgannoaa please properly label each PR and put the issue addressed or fixed in the description. When I look at your open PRs, they all have the same title.
> 
> It seems like I have asked this several times. Please fix immediately.

I refer to your statement - Also for future PRs 
This statement clearly said for the future PR. That is why I have not change the existing name. I just changed now.

**Comment by edwardhartnett** [comment]
@lgannoaa in future, whenever I ask you to do something "in future" then please note that the future starts now.

Which means, do it from that moment forward.

**Comment by lgannoaa** [comment]
Are we clear with this PR? Anymore change do you want?

**Comment by edwardhartnett** [comment]
@lgannoaa I have fixed and will merge.

**Review by edwardhartnett — APPROVED** [review]

**Pull request merged**